packages/ifpack2/src/Ifpack2_ContainerFactory_decl.hpp

```diff
@@ -48,6 +48,7 @@
 #include "Ifpack2_DenseContainer.hpp"
 #include "Ifpack2_SparseContainer.hpp"
 #include "Ifpack2_BandedContainer.hpp"
+#include "Ifpack2_BlockTriDiContainer.hpp"
```

Comment:
Sorry, I'm going to insist on leaving issues orthogonal to this PR out. Too much going on to incorporate more issues.